command/agent.go

```diff
@@ -338,10 +351,43 @@ func (c *AgentCommand) Run(args []string) int {
 
 	ctx := context.Background()
 
+	// Generate the Nomad client.
+	if err := c.agent.GenerateNomadClient(); err != nil {
+		logger.Error("failed to start the nomad client", "error", err)
+		return 1
+	}
+
+	if parsedConfig.HighAvailability.Enable {
+		logger.Info("Running on ha mode")
```

Comment:
I wonder if we also want to log the configuration options passed?
```suggestion
		logger.Info("running in HA mode")
```